src/Medidata.ZipkinTracer.Owin/ZipkinMiddleware.cs

```diff
@@ -1,27 +1,32 @@
 ﻿using System.Threading.Tasks;
+using Medidata.ZipkinTracer.Core;
 using Microsoft.Owin;
 using Owin;
 
-namespace Medidata.ZipkinTracer.Core.Middlewares
+namespace Medidata.ZipkinTracer.Owin
 {
     public class ZipkinMiddleware : OwinMiddleware
     {
         private readonly IZipkinConfig _config;
+        private readonly SpanCollector _collector;
```

Comment:
Looks like _collector is always null here. ZipkinClient will take care the singleton instance of SpanCollector